gsoc/2020/mentors.md

```diff
@@ -5,7 +5,9 @@ layout: plain
 
 ## Full Mentor List (Name, Email, Org)
```

Comment:
make sure you have added all of them

Reply:
Hi @agheata ,
Thank for the review.

please @SergeiML take a look on the comments.

Thanks,
Omar.